buku.py

```diff
@@ -2000,18 +2152,6 @@ def is_bad_url(url):
     return False
 
 
-def is_nongeneric_url(url):
```

Comment:
- [x] Done: can't say for sure why I removed this. Bad merge should be the reason...